### PR TITLE
small refactor for rmAction

### PR DESCRIPTION
**Pull request by junnplus**
Signed-off-by: Ye Sijun <junnplus@gmail.com>

### CODE REVIEW

**Review by fuweid — APPROVED** [review]

**Comment by AkihiroSuda** [comment]
Could you explain what is improved here?

File: cmd/nerdctl/run.go [hunk 1]
```diff
@@ -269,10 +269,6 @@ func setCreateFlags(cmd *cobra.Command) {
 	// #endregion
 }
 
-// runAction is heavily based on ctr implementation:
-// https://github.com/containerd/containerd/blob/v1.4.3/cmd/ctr/commands/run/run.go
-//
-// FIXME: split to smaller functions
```
[the remark this answers is not in the document]
Comment (junnplus) [reply cmd/nerdctl/run.go:275]: Actually, as of https://github.com/containerd/nerdctl/pull/646 `runAction` has separated the code to `createContainer`, and there is a big difference with the link above.

**Comment by junnplus** [comment]
> Could you explain what is improved here?

Just avoid passing the namespace args

File: cmd/nerdctl/run.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (junnplus) [reply cmd/nerdctl/run.go:275]: I make `FIXME` back to `createContainer`

[the remark this answers is not in the document]
Comment (junnplus) [reply cmd/nerdctl/run.go:275]: > and there is a big difference with the link above.

Sorry, I check again it, we need to keep it.

**Review by AkihiroSuda — APPROVED** [review]

**Review by fuweid — APPROVED** [review]

**Pull request merged**